readthedocs/search/parse_json.py

```diff
@@ -1,77 +1,89 @@
 """Functions related to converting content into dict/JSON structures."""
 
-import json
 import logging
+import orjson
```

Comment:
I recommend importing this as `import orjson as json` so that it is easier to swap back to other libraries later if needed.